packages/EasyCodingStandard/packages/ChangedFilesDetector/src/ChangedFilesDetector.php

```diff
@@ -93,7 +93,7 @@ private function storeConfigurationDataHash(string $configurationHash): void
 
     private function fileInfoToKey(SmartFileInfo $smartFileInfo): string
     {
-        return sha1($smartFileInfo->getRealPath());
+        return sha1($smartFileInfo->getRelativeFilePathFromDirectory(getcwd()));
```

Comment:
Seeing this over and over again (not only here), it would be more useful to add `getRelativeFilePathFromCwd()` method to `SmartFileInfo` object

Reply:
Good idea.